query/src/sql/sql_parser.rs

```diff
@@ -229,6 +230,23 @@ impl<'a> DfParser<'a> {
                             Ok(DfStatement::ShowMetrics(DfShowMetrics))
                         } else if self.consume_token("USERS") {
                             Ok(DfStatement::ShowUsers(DfShowUsers))
+                        } else if self.consume_token("FUNCTIONS") {
```

Comment:
It would be nice move these code snippets to a funtion `parse_show_functions` like `parse_show_databases`